Makefile

```diff
@@ -46,14 +46,22 @@ endif
 	bandit -r $(checkfiles)
 	python setup.py check -mrs
 
-test: deps
+test:
 	$(py_warn) TORTOISE_TEST_DB=sqlite://:memory: py.test
 
-_testall:
+test_sqlite:
 	$(py_warn) TORTOISE_TEST_DB=sqlite://:memory: py.test --cov-report=
-	python -V | grep PyPy || $(py_warn) TORTOISE_TEST_DB=postgres://postgres:$(TORTOISE_POSTGRES_PASS)@127.0.0.1:5432/test_\{\} py.test --cov-append --cov-report=
-	$(py_warn) TORTOISE_TEST_DB="mysql://root:$(TORTOISE_MYSQL_PASS)@127.0.0.1:3306/test_\{\}?storage_engine=MYISAM" py.test --cov-append --cov-report=
-	$(py_warn) TORTOISE_TEST_DB="mysql://root:$(TORTOISE_MYSQL_PASS)@127.0.0.1:3306/test_\{\}" py.test --cov-append
+
+test_postgres:
+	python -V | grep PyPy || $(py_warn) TORTOISE_TEST_DB="postgres://postgres:$(TORTOISE_POSTGRES_PASS)@127.0.0.1:5432/test_\{\}?minsize=1&maxsize=20" py.test
```

Comment:
Aha! this misses `--cov-append --cov-report=` at the end, so it clobbers the coverage of the previous run, which is SQLite...